helm/istio/charts/istio_cni/templates/daemonset.yaml

```diff
@@ -24,7 +24,7 @@ spec:
     metadata:
       labels:
         heritage: {{ .Release.Service }}
-        maistra-version: 0.12.0
+        maistra-version: 1.0.0
         release: {{ .Release.Name }}
```

Comment:
this label is added automatically by patch-charts.sh, no?

Reply:
It wasn't but should be, the script hadn't been updated to do that

Reply:
@jwendell you are correct the script does add in the label, the generated version contained two maistra-version labels so the daemonset originally had the 0.12.0 and 1.0.0 versions.  I'll tidy up the overlay version to fix this.